gems/pending/appliance_console/database_replication_master.rb

```diff
@@ -0,0 +1,46 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationMaster < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr master register'.freeze
+
+    def initialize(hash = {})
+      set_defaults
+      super
+    end
+
+    def set_defaults
+      self.host         = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
```

Comment:
Is this `NETWORK_INTERFACE` constant coming from the appliance console? That's a bit strange. Maybe it would be better to ask for "the hostname that will be accessible to the rest of the database cluster" rather than assuming that they want to use an ip address (we could default to the ip though).